swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: ballerina-create-your-first-connector-left-nav-pages-swanlake
+title: Create your first connector with Ballerina
+description: step-by-step guide to create Ballerina connectors using OpenAPI specifications.
+keywords: ballerina, connector, library, REST, API, openAPI, Ballerina Central
+permalink: /learn/create-your-first-connector-with-ballerina/
+active: create-your-first-connector-with-ballerina
+intro: This guide walks you through creating your first Ballerina connector using an OpenAPI specification.
+---
+
+## Introduction
+
+Ballerina is a programming language designed for smooth integration, offering a large library of pre-built connectors. These connectors are special packages that make it easy to communicate with external services, usually through REST APIs. By using connectors, developers can quickly integrate third-party services into their Ballerina applications without having to worry about the technical details of API interactions.
+
+Along with its powerful library ecosystem, Ballerina also allows developers to easily create, share, and manage their own custom connectors. These connectors can be published on Ballerina Central, making them available for the entire community to use and reuse in their projects.
```

Comment:
```suggestion
Along with its powerful library ecosystem, Ballerina also allows developers to easily create, share, and manage their own custom connectors. These connectors can be published on [Ballerina Central](https://central.ballerina.io/), making them available for the entire community to use and reuse in their projects.
```

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-dev-website/commit/7f3e76648db2831c1d956d733e403ad1999cc40d